src/System.Device.Gpio/System/Device/Spi/DataFlow.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Spi
+{
+    /// <summary>
+    /// Defines order in which data is send over the wire.
+    /// </summary>
+    public enum DataFlow
+    {
+        MsbFirst,
```

Comment:
could you please add Xml comments for these here? I know Most/Least significant bits but this may be not obvious for some people

Reply:
@tarekgh most of the people should not change this - when they need to they will likely know what that means :-) Will add though